test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -601,30 +595,36 @@ class test_ucp_tag_nbx : public test_ucp_tag {
                                   const ucp_request_param_t &sparam,
                                   const ucp_request_param_t &rparam)
     {
-        int rcache_count;
-
         if (sparam.op_attr_mask & UCP_OP_ATTR_FIELD_REQUEST) {
             /* Can't send multiple requests for a single user request memory
                handle */
             return;
         }
 
+        UCS_STATS_NODE_DECLARE(stats)
+        stats                 = sender().ucph()->rcache->stats;
+        int rcache_count_prev = UCS_STATS_GET_COUNTER(stats, UCS_RCACHE_GETS);
```

Comment:
`const`

Reply:
you mean on the pointer type? or on the integer?